src/data_sets.h

```diff
@@ -3314,6 +3314,12 @@ typedef struct PACKED
 	/** Magnetometer calibration quality sensitivity threshold. Typical range is 10-20 (10 default) and 1000 to disable mag calibration quality check, forcing it to be always good. */
 	float                   magCalibrationQualityThreshold;
 
+    /** Minimum SNR (4x CN0) used in GPS RTK solution */
+    uint8_t                 gpsMinSnr;
+
+    /** Reserved */
+    uint8_t                 reserved[3];
```

Comment:
I think if we are going to extend this, we should extend it by more than 4 bytes.  Maybe a uint8_t[4], a uint16_t[2] and a uint32_t[2].  This would give us a fair amount of breathing room to accommodate different types for the foreseeable future.

Reply:
Sounds good